hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -1089,15 +1133,17 @@ object LoadVCF {
   ): (Field, (String, Map[String, String]), Boolean) = {
     val id = line.getID
     val isCall = id == "GT" || callFields.contains(id)
-
-    val baseType = (line.getType, isCall) match {
-      case (VCFHeaderLineType.Integer, false) => TInt32()
-      case (VCFHeaderLineType.Float, false) => entryFloatType
-      case (VCFHeaderLineType.String, true) => TCall()
-      case (VCFHeaderLineType.String, false) => TString()
-      case (VCFHeaderLineType.Character, false) => TString()
-      case (VCFHeaderLineType.Flag, false) => TBoolean()
-      case (_, true) => fatal(s"Can only convert a header line with type 'String' to a call type. Found '${ line.getType }'.")
+    val isEntry = line.getKey == "FORMAT"
```

Comment:
might be a bit cleaner to change `entryFloatType` to just `floatType` and change the following line in `parseHeader`:

    val (infoSignature, infoAttrs, infoFlagFields) = headerSignature(infoHeader, callFields, entryFloatType)

to

    val (infoSignature, infoAttrs, infoFlagFields) = headerSignature(infoHeader, callFields, TFloat64())

Reply:
Fixed - see commit 26fc465.

Reply:
I don't think you need this logic anymore -- if you change the info field headerSignature call to use TFloat64, then the float type passed in here is the one you should use.